README.md

```diff
@@ -24,6 +24,18 @@ Further details are given in the associated
 paper (tba)
 and [notebook](./notebook.ipynb).
 
+## License
+
+The model can be used for non-commercial purposes,
+see [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/).
+For commercial usage,
+a license for
+[devAIce](https://www.audeering.com/products__trashed/devaice/)
```

Comment:
Why does the URL contain `products__trashed` and looks like spam?

I checked the following also works:

```suggestion
[devAIce](https://www.audeering.com/devaice/)
```

Reply:
Hehe, yes I was also wondering, but that's the URL you get when you click `Product->deVAice` from the menu. But good to see it also works with the `trashed` part.